internal/attribute/attribute_test.go

```diff
@@ -94,3 +94,27 @@ func TestSliceValue(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkBoolSliceValue(b *testing.B) {
```

Comment:
How did you measure memory use or allocations from this benchmark? It is not reporting memory use.

Reply:
Adding `	b.ReportAllocs()` seems like a better idea as it will always run with that information and reflect the reported measurements of this PR.